helper_image/scripts/copy-out.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+function usage() {
+    echo
+    echo "Verify the md5sums of two files match"
+    echo "This is used after copy_out directives to ensure the copy_in file matches the copy_out file"
+    echo
+    echo "Syntax: copy-out.sh FILE_IN FILE_OUT"
+    echo
+    echo "examples:"
+    echo "copy-out.sh testuser/test.in testuser/test.out"
```

Comment:
Are these relative paths? How does it know where the lustre mount is?

Reply:
They should be absolute and say: `/lus/global/testuser/test.in`